docs/common/Version.xml

```diff
@@ -40,6 +40,6 @@
     structure that we can use bookinfo/date and other elements for other
     purposes.</para>
 
-    <para>☺</para>
+    <para>☺ © Γ∏</para>
```

Comment:
No, that is intentional. That is a non-printing section, which includes a few wingdings as well as copyright, just to verify that the non-standard fonts work.